src/main/java/teammates/ui/webapi/action/BasicFeedbackSubmissionAction.java

```diff
@@ -155,12 +155,12 @@ protected String getRecipientSection(
             String courseId, FeedbackParticipantType recipientType, String recipientIdentifier) {
         switch (recipientType) {
         case INSTRUCTORS:
-        case SELF:
         case NONE:
             return Const.DEFAULT_SECTION;
         case TEAMS:
         case OWN_TEAM:
             return logic.getSectionForTeam(courseId, recipientIdentifier);
+        case SELF:
```

Comment:
So Yup, I am proposing new solution here. 

By right, when the `recipientType` is `SELF`, we should examine the `giverType` instead. If we didn't do this, it will work if an account is either student or instructor but will generate wrong section when the account is both student and instructor.

The fix will not only cover your `if` check but will also fix other problems (`self` (the `self` is feedback creator (instructor)) -> `self`, `team` -> `self`) (P.S. look like we found some bugs).